typescript-operators/main.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-unused-vars */
+
+interface Customer {
+  name: string;
+  details?: {
+    address?: {
+      city: string;
+      street: string;
+      zip: string;
+    };
+    age?: number;
+  };
+}
+
+const v1 = { value: 'Something' };
+const v2 = '';
+const v3 = undefined;
+const v4 = [3, 5, 7, 9];
+const v5: Customer = {
+  name: 'Carl',
+  details: {
+    age: 82,
+  },
+};
+
+console.log(v1 && 'v1 is truthy');
+console.log(v2 && 'v2 is truthy');
```

Comment:
Not quite! The console logs for these are always going to run. We want to write this in a way so that the console logs only execute if `v1` or `v2` is truthy. So there should only be 1 console log total that fires